### DISCUSSION TITLE
Meetings URL broken

### DISCUSSION THREAD

**Discussion by landolfi-us** [code4recovery/12-step-meeting-list#1095] [Q&A] [answered]
Hi all,

We've had the TSML plug-in installed for some time, but in the past few days it's started behaving oddly. Most recently, the URL district10nia.org/meetings stopped working. I'm new to this, but there is no such directory so I'm lost where to go to fix it. Is that something the TSML install is supposed to handle? I noticed that other working installs of TSML have this URL too, so I thought maybe if I uninstalled and reinstalled the plugin, that would fix it, but although a shortcode I have set up shows all meetings, the meetings URL still doesn't. Any suggestions welcome.

**Comment by tim-rohrer** [comment, marked as answer]
A good first step to solve these types of issue is Settings—>Permalinks, and then simply click on Save. That should rebuild all of the Wordpress links.

**Reply by landolfi-us** [reply]
That solved it! For future reference, was it something I did or is this something that might happen from time to time? Either way, thanks!

**Reply by tim-rohrer** [reply]
Tbh, it is hard to say. If you were changing a lot of URLs, or their structure, that can cause it.

**Comment by agwebdevelopment** [comment]
We have the site built in foreign language and few days ago all URL started to include russian language version in the URL, for example:
https://russianspeakingaa.org/%d1%81%d0%be%d0%b1%d1%80%d0%b0%d0%bd%d0%b8%d1%8f/russian-speaking-2/
If I change URL to what is was before, everything works:
https://russianspeakingaa.org/schedule/russian-speaking-2/

I tried the Permanent Link solution above and that did not work. Was trying to fix it for two days and no luck :(

**Reply by joshreisner** [reply]
has the plugin now been removed? earlier i saw it working at `/schedule` on your site, but links would sometimes be forwarded to `/собрания` and then i'd see a 404

by default the plugin url is `/meetings` -- did you customize `$tsml_slug` in your theme? i would see if you could temporarily undo that change and re-save permalinks to see if that fixes it for you

**Reply by agwebdevelopment** [reply]
I would like to go back to using the default, not sure where I can check $tsml_slug
I uninstalled/removed the plugin and reinstalled it and it is still doing the same thing..

**Reply by agwebdevelopment** [reply]
So strange, I added the slug to the the functions.php and I can now get to the individual meeting : https://russianspeakingaa.org/meetings/russian-group-of-brooklyn-men/

But the full list still displays 404 no matter what I try

**Reply by joshreisner** [reply]
hm, did you try setting permalinks again? i think you might need to do that after setting the slug in `functions.php`

**Reply by agwebdevelopment** [reply]
yes, tried it a few times :) so strange this is happening

**Reply by agwebdevelopment** [reply]
OK I pulled the older version of the site and set the variables you mentioned above. It seems to work now. I am just afraid to touch it :)

**Reply by joshreisner** [reply]
i can recreate the problem locally if i set my site language to russian and save permalinks (with no theme customizations). this is true both in the current version of the site and recent versions. what happens is that the site seems to think that meetings should be at `/собрания` but this address is not found

i can fix it by adding `$tsml_slug = "собрания"` to my theme's functions.php, then the meetings page properly shows up at `/собрания` -- this can be anything, are you doing this too? this would explain why your meetings page is `/schedule` -- this is the exact advice we give people here: https://wordpress.org/plugins/12-step-meeting-list/#can%20i%20change%20the%20url%20of%20the%20meetings%20list%3F
